src/briefcase/commands/run.py

```diff
@@ -214,6 +214,13 @@ def add_options(self, parser):
             dest="appname",
             help="The app to run",
         )
+        # Add a command-line option to specify a target simulator/emulator device
```

Comment:
Simulator device handling is already implemented; and the base `run` command can't include anything that is platform specific.